### PR TITLE
Move zk proof tooltip to proofing card

### PR DESCRIPTION
**Pull request by upacyxou**
- move zk proof toolip to proofing card

### CODE REVIEW

File: src/components/proofs/ConnectAccount.tsx [hunk 1]
```diff
@@ -40,14 +43,36 @@ const bottomWrapper = classnames(
...
         <HeaderText accent>First</HeaderText>
         <div className={bottomWrapper}>
           <BodyText center>
-            Connect a wallet with NFTs to create a ZK proof.
+            <span>
+              Connect a wallet with NFTs to create a{' '}
+              <AccentText color="text-accent">
+                <ToolTip position="bottom" fitContainer text={zkProofText}>
+                  <span
+                    className={classnames(
```
Comment (T-Damer) [inline src/components/proofs/ConnectAccount.tsx:60]: We need to put those `classnames` into separate `const`

File: src/components/proofs/ConnectAccount.tsx [hunk 2]
```diff
@@ -40,14 +43,36 @@ const bottomWrapper = classnames(
...
+                      display('inline-flex'),
+                      alignItems('items-center'),
+                      space('space-x-2')
+                    )}
+                  >
+                    <span className={textDecoration('underline')}>
+                      ZK proof
+                    </span>
+                    <span>
+                      <QuestionMark small />
```
Comment (T-Damer) [inline src/components/proofs/ConnectAccount.tsx:70]: Do you need to wrap it into `span`?

File: src/components/proofs/ConnectAccount.tsx [hunk 3]
```diff
@@ -40,14 +43,36 @@ const bottomWrapper = classnames(
...
+    'In your wallet(s), you have NFTs that can point back to your identity (aka, getting doxxed). But with ZK proof, you can verify ownership of NFTs while staying pseudonymous.'
 
   return (
     <div className={walletContainer}>
       <div className={contentWrapper}>
         <HeaderText accent>First</HeaderText>
         <div className={bottomWrapper}>
           <BodyText center>
-            Connect a wallet with NFTs to create a ZK proof.
+            <span>
```
Comment (T-Damer) [inline src/components/proofs/ConnectAccount.tsx:55]: Do we need this `span` wrapper?

File: src/pages/Main.tsx [hunk 4]
```diff
@@ -36,14 +39,16 @@ export default function () {
...
-      <div className={bottomZkHint}>
-        <CardSeparator
-          numberOfLines={1}
-          gradient="secondary-to-transparent"
-          vertical
-        />
-        <ZkProofHint />
-      </div>
+      {account ? (
+        <div className={bottomZkHint}>
```
Comment (T-Damer) [inline src/pages/Main.tsx:43]: How about to start making animations for this one?

File: src/components/proofs/ConnectAccount.tsx (hunk 2 above)
Comment (upacyxou) [reply src/components/proofs/ConnectAccount.tsx:70]: You're right, we could get rid of this

Comment (upacyxou) [reply src/components/proofs/ConnectAccount.tsx:70]: ty for catch

File: src/components/proofs/ConnectAccount.tsx (hunk 3 above)
Comment (upacyxou) [reply src/components/proofs/ConnectAccount.tsx:55]: oh, you're right, we could get rid of this aswell

Comment (upacyxou) [reply src/components/proofs/ConnectAccount.tsx:55]: ty

File: src/components/proofs/ConnectAccount.tsx (hunk 1 above)
Comment (upacyxou) [reply src/components/proofs/ConnectAccount.tsx]: ty, fixed

**Review by T-Damer — APPROVED** [review]

**Review by smyzin — APPROVED** [review]
lgtm

File: src/pages/Main.tsx [hunk 5]
```diff
@@ -36,14 +39,16 @@ export default function () {
...
+          <CardSeparator
+            numberOfLines={1}
+            gradient="secondary-to-transparent"
+            vertical
+          />
+          <ZkProofHint />
```
Comment (T-Damer) [inline src/pages/Main.tsx:49]: Wait, do we need `ZkProofHint` at all?

Comment (upacyxou) [reply src/pages/Main.tsx]: removed

**Pull request merged**